packages/path_provider/path_provider/pubspec.yaml

```diff
@@ -2,7 +2,7 @@ name: path_provider
 description: Flutter plugin for getting commonly used locations on the Android &
   iOS file systems, such as the temp and app data directories.
 homepage: https://github.com/flutter/plugins/tree/master/packages/path_provider
```

Comment:
The homepage also changes, to `https://github.com/flutter/plugins/tree/master/packages/path_provider/path_provider`

Reply:
(you can push this minor change when you come back to this file when doing the federation of _macos, just to conserve your CI/CD luck ;) )

Reply:
Did here :)